core/templates/dev/head/pages/about/About.js

```diff
@@ -19,6 +19,45 @@
 oppia.controller('About', [
   '$scope', 'UrlInterpolationService',
   function($scope, UrlInterpolationService) {
+    var activeTabClass = 'oppia-about-tabs-active';
+    var hash = window.location.hash.slice(1);
+    var visibleContent = 'oppia-about-visible-content';
+
+    var activateTab = function(tabName) {
+      $("a[id='" + tabName + "']").parent().addClass(
+        activeTabClass
+      ).siblings().removeClass(activeTabClass);
+      $('.' + tabName).addClass(visibleContent).siblings().removeClass(
+        visibleContent);
+    };
+
+    if (hash === 'foundation' || hash === 'license') {
+      activateTab('foundation');
```

Comment:
Makes sense, thanks for the suggestion. You mention I can use it in the html, so I can just to ng-click="onTabClick($scope.TAB_ID_FOUNDATION)" is what you're saying? If not, what is the way to pass the name in the html without hardcoding it? @seanlip